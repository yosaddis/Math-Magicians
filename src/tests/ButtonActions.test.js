import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Button from '../components/ButtonActions';

describe('Unit test for button actions component', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(
        <Button
          typeBtn="AC"
          btnOnClick={() => { }}
          primary
          btnClass="btn-clear"
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
