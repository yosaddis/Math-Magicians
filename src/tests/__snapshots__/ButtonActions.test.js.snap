// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit test for button actions component renders correctly 1`] = `
<button
  className="btn-clear"
  onClick={[Function]}
  type="button"
>
  AC
</button>
`;
